src/insert.rs

```diff
@@ -56,7 +56,7 @@ use reqwest::Response;
 use serde_json::{json, Value};
 
 impl SupabaseClient {
-    /// Inserts a new row into the specified table with automatically generated ID for column `id`.
+    /// Inserts a new row into the specified table with a randomly generated ID for column `id`.
```

Comment:
This is not quite correct!
If the `body` provides a value for its primary key then it will be used. If such value is missing then the database will generate one, depending on the table DDL.